### DISCUSSION TITLE
is anyone else having issues using custom css/themes?

### DISCUSSION THREAD

**Discussion by jamiew0w** [upptime/upptime#263] [General] [answered]
no matter what it looks like the assets/custom.css 404s and doesn't show. feel like i've gone through the documentation 20 times and everything's setup properly.

happening to anyone else?

**Comment by AnandChowdhary** [comment, marked as answer]
I think this is because of https://github.com/upptime/upptime/issues/256, there seems to be an issue with moving the `assets` directory. I'll make sure this is fixed today itself!
